lib/output.js

```diff
@@ -597,6 +598,12 @@ function OutputStream(options) {
             return true;
         }
 
+        if (output.option('webkit')
+            && output.parent(0) instanceof AST_PropAccess
+            && output.parent(1) instanceof AST_Assign) {
```

Comment:
> Can the `webkit` output option be hardcoded in `test/jetstream.js` by default?

I am contemplating between `test/mocha/release.js` and `test/jetstream.js`

Hence the discover for the current fireworks - and no, `AST_Assign` does not help. Though I just found out I forgot to swap `phantomjs` from `2.5.0-beta` to `2.1`, so may be there's a new bug?

Rolling back to and re-testing as we speak.